server/views/map/show.jade

```diff
@@ -1,35 +1,41 @@
 extends ../layout-wide
 block content
-        .text-center.map-fixed-header.asterisk-explanation
+        .text-center.map-fixed-header
             p Challenges required for certifications are marked with a *
             .row.map-buttons
                 button.center-block.btn.btn-sm.btn-block.btn-primary.active#showAll Collapse all challenges
+            .row.map-buttons
+                .input-group
+                    input#map-filter.form-control(type="text" placeholder="challenge name")
```

Comment:
According to the w3c spec, "placeholder" is supposed to be representative of the type of thing you're searching for - an example.  We could do something like an existing challenge name, I guess.  I'm certainly open to suggestions.